.github/workflow-templates/cargo-build/action.yml

```diff
@@ -28,15 +29,6 @@ runs:
       run: |
         mkdir -p mold
         curl -L --retry 10 --silent --show-error https://github.com/rui314/mold/releases/download/v1.1.1/mold-1.1.1-$(uname -m)-linux.tar.gz | tar -C $(realpath mold) --strip-components=1 -xzf -
-    # With rustup's nice new toml format, we just need to run rustup show to install the toolchain
-    # https://github.com/actions-rs/toolchain/issues/126#issuecomment-782989659
-    - name: Setup Rust toolchain
```

Comment:
If you need a more recent version of rust, please change the content of the rust-toolchain file instead, we can help you if it imply some changes on rust files :)

Reply:
Removed